controllers/kepler_controller.go

```diff
@@ -422,7 +422,7 @@ func (r *collectorReconciler) ensureDaemonSet(l klog.Logger) (bool, error) {
 		r.daemonSet.Spec.Template.Spec.HostNetwork = true
 		r.daemonSet.Spec.Template.Spec.ServiceAccountName = r.serviceAccount.Name
 		r.daemonSet.Spec.Template.Spec.Containers = []corev1.Container{{
-			Name: "kepler-exporter",
+			Name: "kepler-exporter", //?
```

Comment:
Is this supposed to be r.Instance.Name + "-exporter"?

Reply:
It is the name of the DaemonSet. So far we've been calling it `kepler-exporter`

Reply:
The Daemonset name is r.Instance.Name + "-exporter" - where "-exporter" is a global variable.  For now it doesn't matter if we keep it kepler-exporter since I don't see any need to change, in the future we can change it if it matters. If it isn't a problem, I can remove the //? and commit.

Reply:
as long as it is consistent everywhere it shouldn't matter.